docs/repositories/issues.md

```diff
@@ -98,7 +98,13 @@ To restore an ignored issue, select **Unignore issue** from the options menu:
 
 !!! info "This section applies to GitHub repositories only"
 
-If Codacy detects code patterns that have suggested fixes, a **Fix issues** button appears above the issue list. Click this button to open a pull request including the suggested fixes and targeting the selected branch.
+If Codacy detects code patterns with suggested fixes, a **Fix issues** button appears above the issue list. Click this button to open a modal with a patch that addresses all resolvable issues. Follow the instructions in the modal to apply the patch:
+
+1.  Copy the patch content to your clipboard.
+1.  Create a new branch from your default branch.
+1.  Apply the fixes in your project's root folder, for example with the `pbpaste | patch` command.
+1.  Review the changes to confirm that they're correct.
+1.  Push the new branch to your repository.
 
 ![Fix issues button](images/issues-fix-issues-button.png)
```

Comment:
@claudiacarpinteiro I'm on the fence about updating these screenshots. The procedure described in the UI has 3 steps, but while writing the procedure, I realized it should be 5. I'm avoiding a screenshot to prevent confusing users. Thoughts?

Reply:
I would detach our procedure steps from the steps on the modal. For that, I would suggest that we use only 2 screenshots:

- Current screenshot showing the **Fix issues** button, placed before the procedure steps
- A screenshot of the modal where the user can see the beginning of the patch content and the copy-to-clipboard button, to place right after "Copy the patch content to your clipboard.". Something like this:

<img width="923" alt="image" src="https://github.com/codacy/docs/assets/26439949/d87cbaca-f60a-45ce-933c-e2119781b9a4">

Then we can proceed with our instructions using as many steps as we need (no need to match the modal steps).